pyffi/__init__.py

```diff
@@ -1,13 +1,14 @@
 """
-:mod:`pyffi` --- Interfacing block structured files
-===================================================
+PyFFI --- Interfacing block structured files
```

Comment:
Why drop the module?

Reply:
An experiment. I didn't do it this time.